itou/templates/siae_evaluations/includes/list_item.html

```diff
@@ -46,16 +45,17 @@
             </div>
         </div>
 
-        {% if item.evaluated_siae.reviewed_at and item.state == 'REFUSED' or item.evaluated_siae.reviewed_at and item.state == 'REFUSED_2' %}
-            <div class="row mt-5">
-                <div class="col-md-12">
-                    <h3>Commentaires de la DDETS</h3>
-                    {{ item.labor_inspector_explanation }}
-
-                </div>
-            </div>
+        {% if not item.hide_state_from_siae %}
+            {% with jobapp_state=item.compute_state %}
```

Comment:
Un peu dommage de calculer l'état si `item.evaluated_siae.reviewed_at` est vide, tu ne veux pas du `{% if not item.hide_state_from_siae and item.evaluated_siae.reviewed_at %}` ?